packages/components/src/Notification/Notification.component.js

```diff
@@ -267,7 +278,7 @@ class NotificationsContainer extends React.Component {
 }
 
 const notificationShape = {
-	id: PropTypes.any.isRequired,
+	id: PropTypes.any.isRequired, // each notification must have a unique id
```

Comment:
This comment is not really useful as it describe that the id proptypes is required, just next to the code.
I suggest you to remove it.